### DISCUSSION TITLE
Who's in favor of _ohlc instead of _ochl?

### DISCUSSION THREAD

**Discussion by highfestiva** [highfestiva/finplot#375] [Polls]
finplot has the main function named `candlestick_ochl` instead of `candlestick_ohlc`. It's named after how the columns are internally stored and used for auto-scaling the Y-axis (using the last two columns, same as in `volume_ocv`, which auto-injects a base-line column with only zeroes).

Changing the input format to OHLC will require a little bit of a performance penalty to re-arrange the columns, and also some maintenance to ensure that this transformation works with real-time updates by keeping track of what order columns are expected in for `plot.update_data()` calls, etc.

The Open-Close-High-Low/candlestick_ochl() intuitively has the same order as Open-Close-Volume/volume_ocv() . They also share the same column order in the internal representation when rendering. If changing order of the former, it would make sense to change the order of the latter, including the internal representation. But Open-Volume-Close is utterly nonsensical.

Also ordering of columns in e.g. a MACD would change, at least internally (as it's pretty much the same as the volume plot, i.e. a bar chart with four colors based on a signal).

**Comment by highfestiva** [comment]
The package has 3k downloads every month, so I'd wanna see a few votes before I change this.

**Comment by Hoernchen** [comment]
* dxfeed is ohlc
* polygon is hcl(n)o or ochl

(talking about native delivery order of aggregated bars here)

So as far as the feeds are concerned there is no standardized order anyway, and it doesn't really matter if the plotting part also looks "weird", because it's already weird.
